src/Shared/EventArgsFormatting.cs

```diff
@@ -259,9 +305,17 @@ int threadId
             }
 
             // If the project file was specified, tack that onto the very end.
+            // Check for additional properties that should be outputted as well
             if (projectFile != null && !String.Equals(projectFile, file))
             {
-                format.Append(" [{10}]");
+                if (LogOutputProperties != null && LogOutputProperties.Length > 0)
+                {
+                    format.Append(" [{10}>{11}]");
```

Comment:
Still dithering on the `>`, but still haven't come up with a concrete proposal :(